freeipa.spec.in

```diff
@@ -1,10 +1,10 @@
+# This specfile requires Fedora >= 28 or RHEL > 7.
```

Comment:
It only requires the packages that are available there. I believe anyone is free to try and build this with lower Fedora/RHEL/CentOS versions if they are insane enough to try and build them themselves.
That being said - isn't this information a bit redundant? You just require packages and their versions, if they're not available - bad luck.

Reply:
I tried to be nice and document the fact that the specfile will no longer work on Fedora 27. :)

I'm fine with removing the comment.